components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/api/impl/GroupImpl.java

```diff
@@ -50,6 +50,9 @@
 @SuppressWarnings("NonJaxWsWebServices")
 public class GroupImpl implements Group {
 
+    private static final String GROUP_CANNOT_NULL_MSG = "Group cannot be null.";
```

Comment:
wrong use of grammer for GROUP_CANNOT_NULL_MSG. NULL_GROUP_ERROR_MSG might be a good alternative.